scripts/pipeline-BLOCKCHAIN.sh

```diff
@@ -4,6 +4,8 @@ set -ex
 
 source "${SCRIPT_DIR}/pipeline-COMMON.sh"
 
+export BLOCKCHAIN_SERVICE_INSTANCE=Blockchain-m5
```

Comment:
This should come from the toolchain template UI. If the chaincode template doesn't have a field for the blockchain yet, there's an example in https://github.com/IBM-Blockchain-Starter-Kit/blockchain-toolchain/blob/master/.bluemix/deploy.json (including length validation)